shared-bindings/keypad/Event.c

```diff
@@ -29,32 +29,41 @@
 #include "py/runtime.h"
 
 #include "shared-bindings/keypad/Event.h"
+#include "shared-bindings/supervisor/__init__.h"
 
 //| class Event:
 //|     """A key transition event."""
-//|     def __init__(self, key_number: int=0, pressed: bool=True) -> None:
+//|     def __init__(self, key_number: int=0, pressed: bool=True, timestamp:int=-1) -> None:
```

Comment:
```suggestion
//|     def __init__(self, key_number: int=0, pressed: bool=True, timestamp: Optional[int]=None) -> None:
```
The code and the documentation suggest the default is `None`, not `-1`. Did you mean for `-1` to be used sometimes?